Fix method name (typo)

avoelkl opened the pull request:
### Description
This PR deprecates a current method with a typo and introduces the new method with correct spelling

### Fixed Issues (if relevant)
none

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

avoelkl committed:
:art: Fix typo in method name

dmanners commented on app/code/Magento/Multishipping/Block/Checkout/Overview.php:
```diff
@@ -319,9 +319,18 @@ public function getQuote()
     }
 
     /**
-     * @return mixed
```
I think we might want to keep the return here

dmanners reviewed (changes requested):
Just one small change

avoelkl committed:
Revert ":pencil2: Fix typos in Multishipping and User module"

This reverts commit aa96d022adeff10ad42ab782ca7abb3233f1718d.

avoelkl committed:
:bug: Add phpdoc return type

avoelkl commented:
Hi @dmanners, I updated the PR and re-added the return in the doc.
Please note that I also reverted one commit that accidentally made it into this branch.

dmanners commented:
Thanks @avoelkl I will process this now.

dmanners committed:
Remove unneeded white space

magento-engcom-team commented:
Hi @dmanners, thank you for the review.
[ENGCOM-1675](https://jira.corp.magento.com/browse/ENGCOM-1675) has been created to process this Pull Request

magento-engcom-team commented:
Hi @avoelkl. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.2.6 release.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

ihor-sviziev commented on app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php:
```diff
@@ -5,7 +5,7 @@
 
 /**
- * Multishipping checkout base abstract block
+ * Mustishipping checkout base abstract block
```
Multishipping was correct

ihor-sviziev commented on app/code/Magento/Multishipping/Block/Checkout/Shipping.php:
```diff
@@ -9,7 +9,7 @@
 
 /**
- * Multishipping checkout shipping
+ * Mustishipping checkout shipping
```
Multishipping was correct

ihor-sviziev commented on app/code/Magento/User/Model/Backend/Config/ObserverConfig.php:
```diff
@@ -72,7 +72,7 @@ public function getAdminPasswordLifetime()
 
     /**
-     * Get admin maximum security failures from config
+     * Get admin maxiumum security failures from config
```
Maximum was correct
